pkg/liquidity-mining/contracts/VeBoostV2.vy

```diff
@@ -73,15 +88,42 @@ delegated_slope_changes: public(HashMap[address, HashMap[uint256, uint256]])
 received: public(HashMap[address, Point])
 received_slope_changes: public(HashMap[address, HashMap[uint256, uint256]])
 
-migrated: public(HashMap[uint256, bool])
+# Flag to ensure migration is only performed once
+migrated: public(bool)
+
+# Checked on calls to `boost`, enabling certain Stake DAO and Tetu accounts
+# to create boosts on behalf of their respective lockers
+isApprovedForAll: public(HashMap[address, HashMap[address, bool]])
```

Comment:
Is this really needed though? Just setting the approval to max should be enough without modifying the code, as the allowance is not reduced when used in that case.